src/UI/Component/Glyph/Factory.php

```diff
@@ -699,4 +699,38 @@ public function angry($action = null);
 	 * @return \ILIAS\UI\Component\Glyph\Glyph
 	 */
 	public function attachment($action = null);
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      The Apply Glyph is used to indicate the possibilty of applying changes which the user has made
+	 *      within a control, i.e. a filter.
+	 *   composition: >
+	 *      The Apply Glyph uses the glyphicon-ok.
+	 *   effect: >
+	 *       Upon clicking the page is reloaded immediately with the updated content the control affects. In case of
+	 *       a filter, it means that the entries in a table change in accordance with the filter values.
+	 *
+	 * featurewiki:
+	 *       - https://www.ilias.de/docu/goto.php?target=wiki_1357_Responsive_Table_Filters#ilPageTocA121
+	 *
+	 * rules:
+	 *   usage:
+	 *       1: >
+	 *          The Apply Glyph SHOULD not come without a Reset Glyph and vice versa.
```

Comment:
why?

Reply:
Like in the purpose section said, the Apply Glyph will be used if changes within a control are done. In my opinion, if changes can be applied, there should also be a way to discard them without reloading the whole page. And that's why the Apply Glyph and the Reset Glyph belong together for me. But I can imagine that there could be other use cases for the glyphs, so I use the word "should" and not "must" here.